### ISSUE TITLE
Incorrect heading order on Make an appointment pages

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12084]
## Describe the defect
On VAMC Make an appointment pages ([Example](https://www.va.gov/boston-health-care/make-an-appointment/)), there are 3 headings within the current heading structure that should be changed in order to logically fit the proper structure.

1. "Sign in with a verified account" is currently an H3 and should be an H4 (because it's a child of "Make an appointment online")
2. "How do I talk to someone right now?" is currently an H2 and should be an H3 (because it a child of "Mental health care appointments")
~3. "Specialty care appointments" is currently an H3 and should be an H2 (because it's a sibling of "Primary care appointments" and "Mental health care appointments")~ - no longer true in prod, it's now correctly an h2

## Screenshots
(These are from before the MHV on VA.gov launch changed the widget, so widget copy is different now.) 

![image](https://user-images.githubusercontent.com/106678594/209168703-ba348d4a-57ad-4301-a0a5-3125a7599168.png)

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/12fb676c-97fd-4e57-86ec-f3d45f21614a)

## AC / Expected behavior

- [ ] Investigate if this is still an issue in Next build, and fix it wherever it's present
- [ ] Headers on the "Make an appointment" page follow correct structure
  - [ ] Update "Sign in with a verified account" heading to be an H4
  - [ ] Update "How do I talk to someone right now?" heading to be an H3

## Definition of done
- [ ] Code completed; PR created
- [ ] Review instance created/available; internal team notified to start reviews
- [ ] Automated tests passed 
- [ ] Code review (meets established quality and security standards)
- [ ] PR approved
- [ ] Design review (meets design & UX requirements)
- [ ] A11y review (meets accessibility standards)
- [ ] Functional/feature review (meets acceptance criteria)
- [ ] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated

**Comment by davidmpickett** [comment]
Removing Lovell label as this is a problem on all VAMCs

**Comment by briandeconinck** [comment]
This issue was identified during the Collaboration Cycle Staging Review for Lovell VAMC on 2/28/2023. Applicable Collaboration Cycle labels have been added to this ticket for tracking and dashboard purposes, rather than creating a duplicate ticket.

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team Ticket #[64840](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64840)

**Comment by laflannery** [comment]
Removing CC labels because ticket was recreated

**Comment by Agile6MSkinner** [comment]
@mmiddaugh @jilladams Do we own this widget?

![image](https://github.com/user-attachments/assets/cb0230fd-9502-4bd1-a669-3d2ac7e8f86b)

**Comment by jilladams** [comment]
We currently do! It's the cta-widget in vets-website, and is technically owned by the Public Websites team, but: in this case since it's invoked on a VAMC page, Facilities can def address this header level issue.

**Comment by jilladams** [comment]
@mmiddaugh the copy that will need updating is only in the unauthed state, sounds like? 

In case not, noting: one thing we ran into last time we had to handle these widgets was having the right test users with the correct data states per facility to validate that everything worked correctly after auth, and having the ability to stage changes in an environment where auth would work with test users, in order to verify. (It was a sad time.) If we will need to test auth states, I think we'll need help from someone on auth side (Identity helped last time) or at Health Apps who may have access to test users with the necessary data attributes.

**Comment by jilladams** [comment]
Header level within the "Sign in to view' CTA widget is def a real issue BUT: we've transitioned ownership of the cta-widgets, and as of today Alex Garcia is addressing this on behalf of MHV team: https://dsva.slack.com/archives/C0581MN69TJ/p1738357579748829?thread_ts=1736801646.969559&cid=C0581MN69TJ. 

Not sure if his fix will address everything for the behavior described in the ticket, but once it ships (next week) we can double check.

**Comment by jilladams** [comment]
The old widget was replaced with a new widget, but the same feedback / issue is in play on the new widget. 

Unauthed current state:
![Image](https://github.com/user-attachments/assets/50a86517-b2b8-4add-8825-0801d9818930)

Authed current state: 
![Image](https://github.com/user-attachments/assets/5c7ddfc2-9ae3-4f20-9703-a69024b5891e)


Not urgent, but @Shamiaalam when you have a chance could you review the original report in this ticket, vs. the screenshots above, and confirm whether or not we still have an issue that needs to be fixed? If not, we can close. If so, we can schedule the work.

**Comment by Shamiaalam** [comment]
@jilladams Authed current state don't have H1, as its supposed to follow logical heading level

**Comment by jilladams** [comment]
Did verify otherwise that 2 of the 3 issues Laura reported are still true in prod and should get fixed. Leaving ticket open.

**Comment by Shamiaalam** [comment]
> Did verify otherwise that 2 of the 3 issues Laura reported are still true in prod and should get fixed. Leaving ticket open.

for that issues are we opening a new ticket? or we just leaving this existing ticket?

**Comment by jilladams** [comment]
Leaving this one, and ACs are up to date with what needs to happen.
